samples/languages/java/dacapo-benchmark/java-dacapo-host_config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why not just `host-config.json`?

Reply:
I like making them specific to the application, so from looking at the filename, you have a better understanding what's inside.

Reply:
I find that weird and I don't think we should advocate/recommend that. A folder should be the logical unit that groups things and inside that we have a standard layout. Don't you think?

Reply:
Hmm, I would say that there are two options:

1. We assume that the enclave/host configs should have well-known fixed filenames, i.e. `enclave_config.json` and `host_config.json`. This would be similar to the `Dockerfile` or `Makefile` convention. In that case though, the `--encave-config` and `--host-config` parameters should be optional and only needed if a user (for whatever reason) wants to override the filenames. This would be similar to how `docker` or `make` work -- otherwise we are forcing users to provide redundant options with each invocation.

2. We don't assume that there is a single approved filename and instead require users to be explicit (as it is now).

As you can tell, I'm in favour of (2), as I'm not a big fan of "magic" filenames that people must remember. That said, if people prefer (1), I'm also ok with that.